pkg/storage/tsdb/expanded_postings_cache.go

```diff
@@ -374,6 +379,7 @@ func (c *fifoCache[V]) getPromiseForKey(k string, fetch func() (V, int64, error)
 	loaded, ok := c.cachedValues.LoadOrStore(k, r)
 
 	if !ok {
+		c.metrics.CacheMiss.WithLabelValues(c.name, "miss").Inc()
```

Comment:
Reason is `miss`? But the metric itself is already miss

Reply:
yeah... but idk a better reason in this case, haha what do you recommend?